fenix/app/src/main/java/org/mozilla/fenix/compose/button/Button.kt

```diff
@@ -47,15 +50,18 @@ private fun Button(
     backgroundColor: Color,
     modifier: Modifier = Modifier,
     icon: Painter? = null,
+    iconModifier: Modifier = Modifier,
     tint: Color,
     onClick: () -> Unit,
+    enabled: Boolean = true,
```

Comment:
Let's reorder the `enabled` to be after the `modifier` parameter. We want to keep the lambda/content parameters as the last parameter so that we can `Button { onButtonClick() }` or `Button { Icon() }` if it was a `content` parameter.

Reply:
Still looking for this change

Reply:
Done